container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/docker_kurtosis_backend_enclave_functions.go

```diff
@@ -328,13 +328,6 @@ func (backend *DockerKurtosisBackend) DestroyEnclaves(
 				enclaveUuid,
 			)
 		}
-		if _, found := backend.serviceRegistrations[enclaveUuid]; found {
```

Comment:
I remove this condition because it doesn't make any sense, due to the `backend.serviceRegistrations` was always `0` on Engine.